src/helpers/priceHelper.js

```diff
@@ -44,18 +49,18 @@ const reduceIngredient = (addedIngredients, ingredient) =>
           ? {
               ...ingred,
```

Comment:
ingredient. Do not user short names. You can do it if you use several words as variable name, for example: AscSortedEmpList